ui/shared/components/panel/variants/Variants.jsx

```diff
@@ -111,7 +112,11 @@ const Variant = React.memo(({ variant, isCompoundHet, mainGeneId, linkToSavedVar
         {isCompoundHet && <VariantIndividuals variant={variant} isCompoundHet />}
         {isCompoundHet && showReads}
       </Grid.Column>
-      <Grid.Column><Annotations variant={variant} /></Grid.Column>
+      <Grid.Column>
+        <Annotations variant={variant} />
+        <VerticalSpacer height={12} />
```

Comment:
the `AcmgModal` component should be responsible for handling spacing, so we don't have unnecessary padding if it isn't shown